gio/src/auto/functions.rs

```diff
@@ -181,9 +181,9 @@ pub fn content_type_get_symbolic_icon(type_: &str) -> Icon {
 #[doc(alias = "g_content_type_guess")]
 pub fn content_type_guess(
     filename: Option<impl AsRef<std::path::Path>>,
-    data: &[u8],
+    data: Option<&[u8]>,
```

Comment:
Needs rebasing as this is now manually implemented. Can be autogenerated again :)

Reply:
It was only reverted in the 0.20 branch. main still has it and should have it with the `Option`.